cloud/c#/TokenConsumer.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Pulsar.Client.Api;
+
+namespace CsharpExamples
+{
+    internal class JWT
+    {
+        internal static async Task Consumer()
+        {
+            const string serviceUrl = "{{ SERVICE_URL }}";
+            const string subscriptionName = "test-sub";
+            var topicName = "persistent://public/default/test-topic";
+            var token = "{{ AUTH_PARAMS_TOKE }}";
```

Comment:
```suggestion
            var token = "{{ AUTH_PARAMS_TOKEN }}";
```
As pointed out by Huanli in [this comment](https://github.com/streamnative/sn-docs/pull/236#discussion_r1166542763) - not sure if it is intentional or a typo? There are multiple occurrences in the commit.